src/utils/ChainPatrolApiClient.ts

```diff
@@ -0,0 +1,204 @@
+import axios from "axios";
+import { env } from "../env";
+
+export type DiscordGuildStatusType = {
+  guildId: string;
+};
+
+export type DiscordGuildStatusResponseType = {
+  connected: boolean;
+  guildId?: string;
+  channelId?: string;
+  organizationName: string;
+  organizationUrl: string;
+};
+
+export enum AssetType {
+  URL = "URL",
+  PAGE = "PAGE",
+  ADDRESS = "ADDRESS",
+  DISCORD = "DISCORD",
+  TWITTER = "TWITTER",
+}
+
+export type ReportCreateAssetType = {
+  content: string;
+  status: AssetStatus;
+  type?: AssetType;
+};
+
+export type ReportCreateType = {
+  assets: ReportCreateAssetType[];
+  attachmentUrls: string[];
+  contactInfo?: string;
+  description: string;
+  discordGuildId?: string;
+  organizationSlug?: string;
+  title: string;
+};
+
+export type ReportCreateOrganizationResponseType = {
+  name: string;
```

Comment:
This was inferred from the usage within this repo, [Documentation](https://chainpatrol.io/docs/external-api/report-create#response) is actually missing some info

Reply:
Added the missing fields and created a PR internally to fix docs